src/tests/test_parse.py

```diff
@@ -121,7 +144,7 @@ def test_parse_early_newlines():
 def test_parse_connect_shovel_over_newlines():
     assert parse("\nsqlite://\ndest\n<<\nSELECT *\nFROM work", empty_config) == {
         "connection": "sqlite://",
-        "sql": "SELECT *\nFROM work",
+        "sql": "\nSELECT *\nFROM work",
```

Comment:
Why did we change it?

Reply:
Great point, I had earlier added .strip() in the func that strips away empty spaces and new line which did't require to edit this piece of code by it failed other test cases(5 int total) especially in test_command.py: 
one of the test cases
```
def test_parsed_sql_when_using_file(ip, sql_magic, tmp_empty):
    Path("query.sql").write_text("SELECT * FROM author")
    cmd = SQLCommand(sql_magic, ip.user_ns, "--file query.sql", "")

    assert cmd.parsed == {
        "connection": "",
        "result_var": None,
        "return_result_var": False,
        "sql": "SELECT * FROM author\n",
        "sql_original": "SELECT * FROM author\n",
    }

    assert cmd.connection == ""
    assert cmd.sql == "SELECT * FROM author\n"
    assert cmd.sql_original == "SELECT * FROM author\n"
```

cause they require to have newline